grapple/types/streamfield.py

```diff
@@ -22,7 +22,7 @@
 class GenericStreamFieldInterface(Scalar):
     @staticmethod
     def serialize(stream_value):
-        return stream_value.stream_data
+        return stream_value.raw_data
```

Comment:
@zerolab I might lack a bit of knowledge regarding StreamField representation. 
Can this remain as is or should I update as well ?

Reply:
`raw_data` is good.

Reply:
saying that, this will fail for anything < 2.12 (and we have a `wagtail>=2.5, <2.14` constraint).

We want to support 2.11+, at the very least, so a good old try/except should do the trick here

Reply:
Silly me 🤦 
Thanks for the review, I will update.

Reply:
@fabienheureux you can now merge the latest code from the `main` branch which contains an updated CI pipeline that runs tests on multiple Wagtail versions (2.9, 2.10, 2.11, 2.12).

After merging you can update the [wagtail versions list](https://github.com/GrappleGQL/wagtail-grapple/blob/main/.github/workflows/ci.yml#L32) to have version 2.13 as well!

Reply:
@fabienheureux let me know if you have any questions about the previous comment 😉

Reply:
Thanks for the suggestion, no question it was perfectly clear (just lacked a bit of free time to work on this PR lately).

Reply:
Test pass with all wagtail versions, I checked and it seems that `raw_data` property exists since a few Wagtail versions, for example here in 2.11 https://github.com/wagtail/wagtail/blob/1061caa5eff53f450520284b618e615fc2a809c8/wagtail/core/blocks/stream_block.py#L496

Reply:
After a bit of debugging, I don't get where this code is used. 
Any idea ?

Reply:
```suggestion
        try
            return stream_value.raw_data
        except AttributeError:
            return stream_value.stream_data
```